tests/framework/installer/ceph_manifests.go

```diff
@@ -1675,6 +1676,52 @@ subjects:
 
 // GetRookCluster returns rook-cluster manifest
 func (m *CephManifestsMaster) GetRookCluster(settings *ClusterSettings) string {
+	if settings.UsePVCs {
+		return `apiVersion: ceph.rook.io/v1
+kind: CephCluster
+metadata:
+name: ` + settings.Namespace + `
+namespace: ` + settings.Namespace + `
+spec:
+cephVersion:
+  image: ` + settings.CephVersion.Image + `
+  allowUnsupported: ` + strconv.FormatBool(settings.CephVersion.AllowUnsupported) + `
+dataDirHostPath: ` + settings.DataDirHostPath + `
+network:
+  hostNetwork: false
+mon:
+  count: ` + strconv.Itoa(settings.Mons) + `
+  allowMultiplePerNode: true
```

Comment:
In this test we should also use PVCs for mons.

Reply:
done